index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Module 1 </title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    
+    <div class="brown">
+        <section><ul><li>Portoflio</li>
+            <li>About</li>
+            <li>Conduct</li></ul>
+        </section>
+        <header class="header"> 
```

Comment:
- [ ] You have done a great job wrapping your heading section. However, as this is a block of code, this would be semantically correct to use a ```<section>``` instead of using ```header>``` as you have used. If you may be having doubts, have a look at [this](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) resource to learn when to or not to use the ```<section>``` tags.